installer.sha

```diff
@@ -0,0 +1 @@
+228635e5b87925773b9b4e12ceb89ec8cecadab3
```

Comment:
Fix this SHA. This commit is in the `master` branch, and not in the `release-1.3` branch.
You must use a SHA in the `release-1.3` branch here.

Reply:
Done.